.vscode/tasks.json

```diff
@@ -122,7 +122,7 @@
     },
     {
         "label": "run tests in current file (netcoreapp3.1)",
```

Comment:
boy, this task has been sitting unused for a little while :P

Reply:
@RikkiGibson Should I remove in this PR?